HLTriggerOffline/SUSYBSM/python/SUSYBSM_HLT_Electron_BJet_cff.py

```diff
@@ -10,7 +10,7 @@
   TriggerPath = cms.string('HLT_Ele10_CaloIdM_TrackIdM_CentralPFJet30_BTagCSV_p13_v'),
   TriggerFilterEle = cms.InputTag('hltSingleEle10CaloIdTrackIdVLDphiFilter', '', 'HLT'), #the last filter in the path hltSingleEle10CaloIdTrackIdVLOneOEMinusOneOPFilterRegional
   TriggerFilterJet = cms.InputTag('hltCSVFilterSingleEle10', '', 'HLT'), #the last filter in the path
-  PtThrJet = cms.untracked.double(40.0),
+  PtThrJet = cms.untracked.double(30.0),

```

Comment:
perhaps it is worth putting this constant in one spot so that next time it changes you don't have to change it in 10 places? @olivito